docs/reference-guide/cookieplone-make-commands.md

```diff
@@ -0,0 +1,96 @@
+---
+myst:
+  html_meta:
+    "description": "Cookieplone make commands"
+    "property=og:description": "Cookieplone make commands"
+    "property=og:title": "Cookieplone make commands"
+    "keywords": "Cookieplone, make, commands"
+---
+
+# Cookieplone `make` commands
+
+This guide explains all the `make` commands found in a cookieplone project.
+
+## Frontend
+
+The Makefile at the root of your project invokes commands in frontend/Makefile. Here are excerpts from frontend/Makefile to show details of the frontend make commands.
+
+`frontend-install`
+
+The command `make frontend-install`:
+
+-   Invokes the target `make install` in `frontend/Makefile`.
+    -   This installs the add-ons in the development environment.
+
```

Comment:
Let's use a definition list for each target. It's cleaner and easier to read. Here's one example. I'll leave the rest for you to perform.

```suggestion
`frontend-install`
:   Invokes the target `make install` in `frontend/Makefile`.
    This installs the add-ons in the development environment.

```